C/plugins/north/PI_Server_V2/plugin.cpp

```diff
@@ -41,6 +41,11 @@ using namespace rapidjson;
 				"\"description\": \"The producer token that represents this FogLAMP stream\", " \
 				"\"type\": \"string\", \"default\": \"omf_north_0001\", " \
 				"\"order\": \"2\" }, " \
+			"\"source\": {" \
```

Comment:
This should be an enumeration rather than a string

Reply:
you are right, fixing.

Reply:
fixed